prow/scripts/cluster-integration/serverless-integration-k3s.sh

```diff
@@ -166,11 +62,23 @@ sleep 60
 SERVERLESS_CHART_DIR="${KYMA_SOURCES_DIR}/resources/serverless"
 job_name="k3s-serverless-test"
 
+if [[ ${INTEGRATION_SUITE} == "git-auth-integration" ]]; then
+  echo "--> Fetching Serverless k3s-tests"
+  git clone https://github.com/kyma-project/kyma "${KYMA_SOURCES_DIR}"
+
+  job_name="k3s-serverless-nightly-test"
+fi
+
+VALUES="-f ${SERVERLESS_CHART_DIR}/values.yaml"
+# check for test overrides.
+if [[ -e "${SERVERLESS_OVERRIDES_DIR}/integration-overrides.yaml" ]]; then
+  VALUES+=" -f ${SERVERLESS_OVERRIDES_DIR}/integration-overrides.yaml"
+fi
 
-helm install serverless-test "${SERVERLESS_CHART_DIR}/charts/k3s-tests" -n default -f "${SERVERLESS_CHART_DIR}/values.yaml" --set jobName="${job_name}"
+#shellcheck disable=SC2086
+helm install serverless-test "${SERVERLESS_CHART_DIR}/charts/k3s-tests" -n default ${VALUES} --set jobName="${job_name}"
```

Comment:
I retriggered pjtester, but I coulnd't find test `git-auth-integration`.
I only found: k3s: `serverless-integration` and `serverless-git-auth-integration` which run the same suite: `serverless-integration`, which is the default one.

I noticed that you didn't pass the override for testSuite while installing the chart.